### ISSUE TITLE
Need clarification of difference between ' and `

### ISSUE THREAD

**Issue by stimsong** [learn-co-curriculum/skills-based-js-intro-to-functions#33]
Hello,

This lesson needs some sort of clarification, note, highlight, or something to point out the difference between ' (a single quote) and ` (not sure what it's called). I've copied a few lines of the lesson where it first shows up (surrounding "Hello, ${firstName}!")

I thought it was a single quote and spent about an hour trying to figure out what I was doing wrong, lol. I wanted to figure it out myself by research and trying things out before I posted a question. I finally noticed the slight difference between ' and ` (this is on the ~ (tilde) key on my keyboard). A mention of this may help other students :)

SAYING HELLO, AGAIN
We found it kind of tiresome to say hello to everyone individually above — what if we told you there's a better way?

function sayHelloTo(firstName) {
  console.log(`Hello, ${firstName}!`)
}
 
sayHelloTo('Isabel') // "Hello, Isabel!"
sayHelloTo("Jane") // "Hello, Jane!"
sayHelloTo("R2-D2") // "Hello, R2-D2!"

Thanks,
Greg

P.S. I'll probably never forget about ` now! That's the great thing about figuring it out yourself :)

**Comment by AwkwardGestures** [comment]
Definitely! This post actually answered a question I had when I couldn't get the console to recognize ${firstName} as an argument object. Thanks stimsong!

**Comment by ghost** [comment]
Wow!  The time I've spent trying to figure out what I was doing wrong!!   Yes they need to explain that a new "backquote" ` .  As most of us are new it's the smallest little detail that is extremely hard to notice to an untrained eye.  Over and over again I was putting in sing quote.  This should really be brought up as a big flag.

**Comment by ascotttoney** [comment]
Damn. Thank you!
